cloud/azure/functions/detectors-functions.tf

```diff
@@ -0,0 +1,108 @@
+resource "signalfx_detector" "heartbeat" {
```

Comment:
Add `threshold` function & use `lasting` function instead of aggregation function for Function detectors.